Tasks/AzureRmWebAppDeploymentV4/deploymentProvider/PublishProfileWebAppDeploymentProvider.ts

```diff
@@ -103,6 +105,16 @@ export class PublishProfileWebAppDeploymentProvider implements IWebAppDeployment
         process.env.PATH = this.origEnvPath;
     }
 
+    private OverrideSetParametersFilePath(): void {
+        // Override set parameters file path if SetParametersFile input is provided
+        // The deploy.cmd file uses '_DeploySetParametersFile' environment variable to determine the value of the set parameters file path
+        // If SetParametersFile input is not provided, its value is set as default working directory because it is a FilePath input type.
+        if (!!this.taskParams.SetParametersFile && this.taskParams.SetParametersFile !== tl.getVariable('System.DefaultWorkingDirectory')) {
```

Comment:
Can the value of  `SetParametersFile` be zero or an empty string, in that case `!!this.taskParams.SetParametersFile` check will fail?
Do we have any util method that we can use such as lodash' isNil ?

Reply:
it will not be 0 as an integer value. it can only be string and can also be an empty string. `!!this.taskParams.SetParametersFile` will return false for empty string which is expected . we have not used any such utility till now.